src/fiskaltrust.Launcher/ProcessHost/ProcessHostMonarch.cs

```diff
@@ -72,6 +75,11 @@ protected override async Task ExecuteAsync(CancellationToken cancellationToken)
                 _lifetime.ServiceStartupCompleted();
             }
 
+            if (_hosts.Count == 0)
+            {
+                await new TaskCompletionSource().Task; // wait for shutdown
```

Comment:
Without this the launcher shuts down with a TaskCancelled exception if it's started with nothing to host (because the monarch background service edits) ^^ And I want to test the selfudpate with an empty configuration so we need this

Reply:
Interesting 😅 could you maybe explain this in a code comment as well? I'm afraid we'll forget this otherweise.